OCPBUGS-55829: SELinuxd image mapping regex

Vincent056 opened the pull request:
Fixed regex issue on OCP4.19 where /etc/os-release has been changed to use RHEL version instead of 4.xx Patterns for CoreOS versions in multiple manifests and tests

clone was removed in https://github.com/openshift/security-profiles-operator/commit/a0ad03d2cdc74d60059d602058a8df9dbb71d02a, but it is needed for selinuxd to work on RHEL8 node

Vincent056 committed:
OCPBUGS-55829: SELinuxd image mapping regex patterns for CoreOS versions in multiple manifests and tests

Vincent056 committed:
Fix replace selinuxd image regex to Support  Red Hat Enterprise Linux release formate

rhmdnd commented:
Looks good, EC testing is failing on unrelated issues with the image labels:

```
Reason: The "description" label should not be inherited from the parent image
Reason: The "summary" label should not be inherited from the parent image
```

Vincent056 committed:
Add 'clone' capability to security profiles operator configmap

This is to fix the failures on OCP 4.12 rhel8 node
```
oc logs pod/spod-985gr -c security-profiles-operator
runtime/cgo: pthread_create failed: Operation not permitted
SIGABRT: abort
PC=0x7f11d2de2a6c m=0 sigcode=18446744073709551610
```

rhmdnd commented on bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml:
```diff
@@ -193,6 +193,7 @@ data:
             "capget",
             "capset",
             "chdir",
+            "clone",
```
Interesting - did something change in 4.12 where this is required, now?

Vincent056 replied:
this was there in the past, but was removed in https://github.com/openshift/security-profiles-operator/commit/a0ad03d2cdc74d60059d602058a8df9dbb71d02a

rhmdnd reviewed (approved):
/lgtm
